src/shared/utils/helpers/num-to-si.ts

```diff
@@ -6,5 +6,8 @@ const SHORTNUM_SI_FORMAT = new Intl.NumberFormat("en-US", {
 });
 
 export default function numToSI(value: number): string {
+  if (value < 10000) {
```

Comment:
I'm not a fan of this hack. I would consider using 2 instead of 3 significant figures tho when using the shortnum versions.